ext/authx/Civi/Authx/AuthxRequestBuilder.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+namespace Civi\Authx;
+
+use GuzzleHttp\Psr7\AppendStream;
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Utils;
+
+/**
+ * (TEST HELPER/EXPERIMENTAL)
```

Comment:
why doesn't it live in the `tests` directory?

Reply:
Yeah, it's a test helper, but the helper is used by tests in two extensions (`authx` and `afform`). Classloading gets weirder if you try share code from the `tests/` folder. (*Slightly different, but similar idea - note how newer helpers tend to go in `civicrm-core:Civi/Test/` rather than `civicrm-core:tests/phpunit/CiviTest/`. It's because the class-loading is clearer.*)

Reply:
:+1: